### DISCUSSION TITLE
Setting nbconvert citation syntax for Jupyter Notebooks

### DISCUSSION THREAD

**Discussion by saf-dmitry** [retorquere/zotero-better-bibtex#2680] [Q&A] [answered]
My question to the community is how can I customize the Quick Copy format to produce citations using syntax compatible with the [nbconvert](https://nbviewer.org/github/jupyter/nbconvert-examples/blob/master/citations/Tutorial.ipynb) tool for Jupyter Notebooks:

```
<cite data-cite="granger2013">(Granger, 2013)</cite>
```

Here, `grander2013` should be the cite key generated by BBT.

**Comment by retorquere** [comment, marked as answer]
If you open an issue, I can add it to the quick copy exporter

**Comment by saf-dmitry** [comment]
Thank your for your help. I've opened an issue (#2681).
